templates/beego/files/routers/router.go

```diff
@@ -0,0 +1,11 @@
+package routers
+
+import (
+	"github.com/platformsh/template-beego/controllers"
+	"github.com/astaxie/beego"
+)
+
+func init() {
+    beego.Router("/", &controllers.MainController{})
+		beego.Router("/test", &controllers.MySQLTestController{})
+}
```

Comment:
If I'm following correctly, this means the import statement in main.go automatically triggers this init() to run, and that's what does the wiring to configure Beego, correct?  Because otherwise I don't see how the configuration happens.  Is this how Beego documents you should do it?